lib/account-handler.js

```diff
@@ -1,133 +1,109 @@
 const {findProcess,stopProcess,startProcess} = require('./process-handler')
-const {removeParsecUserFile,createAccount,checkNickname,removeAccount,switchAccountData,getAccountList,checkUserFile} = require('./account-data')
-const sleep = require('sync-sleep')
-class ProcessException extends Error{
-    constructor(message) {
-        super(message);
-        this.name = 'ProcessException';
-    }
-}
+const {checkNickname,removeAccount,switchAccountData,getAccountList } = require('./account-data')
+const {error} = require("./error");
+const pWaitFor  = require('p-wait-for')
+
+
 async function stopParsec(){
-    const output = await findProcess('parsecd')
-    const op2 = await findProcess('pservice')
-    //console.log(JSON.stringify(op2))
-    //console.log(JSON.stringify(output))
+    //todo logger
 
 
-    if (op2.success){
+    if (await findProcess('pservice')){
         const stopService = await stopProcess("pservice.exe")
-        if (!stopService.success){
-            throw new Error(stop.message)
+        if (!stopService){
+            return error.STOP_FAILED
+            //todo logger
         }
     }
-    if(output.success){
+    if(await findProcess('parsecd')){
         const stop = await stopProcess("parsecd.exe")
-        if (!stop.success){
-            throw new Error(stop.message)
+        if (!stop){
+            return error.STOP_FAILED
+            //todo logger
         }
-        //console.log(JSON.stringify(stop))
     }
```

Comment:
as stopProcess is always used with findProcess maybe its a better idea to directly use find inside stopProcess

Reply:
done